cli/server.go

```diff
@@ -1406,15 +1405,27 @@ func configureTLS(tlsMinVersion, tlsClientAuth string, tlsCertFiles, tlsKeyFiles
 		return nil, nil //nolint:nilnil
 	}
 
-	err = configureCAPool(tlsClientCAFile, tlsConfig)
```

Comment:
Not sure about this removal, it seems to me the client CA is for the server to verify client certificates and is not something the client itself needs but I think I might be missing something.